doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -290,6 +290,8 @@ Bluetooth samples
 
     * :ref:`peripheral_hids_mouse`
 
+  * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target in the :ref:`power_profiling` sample.
```

Comment:
After all, this might be better be its own entry like:

* :ref:`power profiling` sample:

  * Added support for the ...